packages/editor/src/toolbar/popups/color-picker.tsx

```diff
@@ -220,6 +268,17 @@ export function ColorPicker(props: ColorPickerProps) {
                 }
               }}
               onClick={() => {
+                if (trashMode) {
```

Comment:
Deleting custom colors should be done on the same level as saving custom colors. How do we know for certain that custom colors are always stored in `config`? We don't know what the `onSave` function did. Hence, it is better to leave it up to the upper layer to decide what to do.